cvat/apps/dataset_manager/views.py

```diff
@@ -97,6 +97,40 @@ def clear_export_cache(task_id, file_path, file_ctime):
         log_exception(slogger.task[task_id])
         raise
 
+def backup_task(task_id, output_path):
+    try:
+        db_task = Task.objects.get(pk=task_id)
+
+        cache_dir = get_export_cache_dir(db_task)
+        output_path = osp.join(cache_dir, output_path)
+
+        task_time = timezone.localtime(db_task.updated_date).timestamp()
```

Comment:
Do we need to adjust time for clearning_job somewhere here? I believe we have multiple races here. Several users can try to backup the same task in parallel. Clearning job can remove our file after all our successful checks.

Races are fine if they don't lead to unexpected results.